bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,28 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
```

Comment:
right, so touching the cache is out of scope, right? I spent quite some time to get this caching topic perfect. It reduces build time of test down to roughly 2 minutes.

-> caching the eggs directory is not enough, you want to cache downloads as well.

Also I simplified to pass this information about the caches explicit to buildout as you see on current master. Please sync the PR with it.